test/repository.spec.js

```diff
@@ -349,11 +349,12 @@ describe('Repository', function() {
 
       it('should write to repo', function(done) {
          remoteRepo.writeFile('master', fileName, initialText, initialMessage, assertSuccessful(done, function() {
-            remoteRepo.getContents('master', fileName, 'raw', assertSuccessful(done, function(err, fileText) {
+            wait().then(() => remoteRepo.getContents('master', fileName, 'raw',
```

Comment:
Sometimes the `writeFile` succeed even though github didn't really create the file yet and the `getContents` call fails with 404. `Wait` is there to give time to Github to settle down